forum/communications/consumers.py

```diff
@@ -0,0 +1,48 @@
+from channels.generic.websocket import AsyncJsonWebsocketConsumer
+from users.models import CustomUser
+
+
+class ChatConsumer(AsyncJsonWebsocketConsumer):
+    async def connect(self):
+        self.room_name = self.scope["url_route"]["kwargs"]["room_name"]
+        self.room_group_name = f"chat_{self.room_name}"
+
+        await self.channel_layer.group_add(
+            self.room_group_name, self.channel_name
+        )
+
+        await self.accept()
```

Comment:
Consider adding authentication checks to ensure that only authenticated users can connect.

Reply:
We are using middleware to ensure that the user is authenticated, so it's unnecessary to verify it in consumers.